unimath-symbols.ltx

```diff
@@ -163,14 +173,19 @@ package.
 Use this document to find the command name or the Unicode glyph slot for a symbol that you wish to use.
 The following fonts are shown: (with approximate symbol counts)
 \begin{itemize}
-\item[M] \mathversion{lm} $\mathup{Latin\ Modern\ Math}$ (\ref{count:lm})
-\item[X] \mathversion{xits} $\mathup{XITS\ Math}$ (\ref{count:xits})
-\item[S] \mathversion{stix} $\mathup{STIX\ Math}$ (\ref{count:stix})
-\item[C] \mathversion{cambria} $\mathup{Cambria\ Math}$ (\ref{count:cambria})
-\item[A] \mathversion{asana} $\mathup{Asana\ Math}$ (\ref{count:asana})
-\item[P] \mathversion{pagella} $\mathup{TeX\ Gyre\ Pagella\ Math}$ (\ref{count:pagella})
-\item[D] \mathversion{dejavu} $\mathup{DejaVu\ Math\ TeX\ Gyre}$ (\ref{count:dejavu})
-\item[E] \mathversion{euler} $\mathup{Neo\ Euler}$ (\ref{count:euler})
+	\item[M] \mathversion{lm} $\mathup{Latin\ Modern\ Math}$ (\ref{count:lm})
+	\item[X] \mathversion{xits} $\mathup{XITS\ Math}$ (\ref{count:xits})
+	\item[S] \mathversion{stix} $\mathup{STIX\ Math}$ (\ref{count:stix})
+	\item[S2] \mathversion{stixTwo} $\mathup{STIX\ Two\ Math}$ (\ref{count:stixTwo})
+	\item[C] \mathversion{cambria} $\mathup{Cambria\ Math}$ (\ref{count:cambria})
+	\item[A] \mathversion{asana} $\mathup{Asana\ Math}$ (\ref{count:asana})
+	\item[P] \mathversion{pagella} $\mathup{TeX\ Gyre\ Pagella\ Math}$ (\ref{count:pagella})
+	\item[D] \mathversion{dejavu} $\mathup{DejaVu\ Math\ TeX\ Gyre}$ (\ref{count:dejavu})
+	\item[B] \mathversion{bonum} $\mathup{Bonum\ Math\ TeX\ Gyre}$ (\ref{count:bonum})
+	\item[S] \mathversion{schola} $\mathup{Schola\ Math\ TeX\ Gyre}$ (\ref{count:schola})
+	\item[R] \mathversion{retermes} $\mathup{Retermes\ Math\ TeX\ Gyre}$ (\ref{count:retermes})
```

Comment:
Your naming of the TeX Gyre fonts is a bit inconsistent.  I propose
```
	\item[P] \mathversion{pagella} $\mathup{TeX\ Gyre\ Pagella\ Math}$ (\ref{count:pagella})
	\item[D] \mathversion{dejavu} $\mathup{TeX\ Gyre\ DejaVu\ Math}$ (\ref{count:dejavu})
	\item[B] \mathversion{bonum} $\mathup{TeX\ Gyre\ Bonum\ Math}$ (\ref{count:bonum})
	\item[S] \mathversion{schola} $\mathup{TeX\ Gyre\ Schola\ Math}$ (\ref{count:schola})
	\item[T] \mathversion{retermes} $\mathup{TeX\ Gyre\ Termes\ Math}$ (\ref{count:retermes})
```
I have also changed “Retermes” into “Termes” and the corresponding symbol from “R” to “T”.